opentelemetry/src/test/scala/zio/telemetry/opentelemetry/tracing/TracingTest.scala

```diff
@@ -123,7 +123,7 @@ object TracingTest extends ZIOSpecDefault {
               .flatMap(_.getAttributes.asMap().asScala.toList.map(x => x._1.getKey -> x._2.toString)),
             hasSubset(List("exception.message" -> "some_error", "exception.type" -> "java.lang.RuntimeException"))
           )
-          val errorMapper                       = ErrorMapper[Throwable](_ => StatusCode.ERROR, Some(identity))
+          val errorMapper                       = ErrorMapper[Throwable]({ case _ => StatusCode.ERROR }, Some(identity))
```

Comment:
A small thing - we don't `Some(identity)` anymore since we made this parameter None by default. 
Btw, nice catch! It is strange why the difference in the syntax causes the compiler to fail.

Reply:
if I set None then I can't catch error and test will be failed

Reply:
Ah sorry, you're right!